README.rst

```diff
@@ -432,6 +435,117 @@ view (only when using HTTP POST).
 The signal is emitted just before a successful response is returned,
 it is not sent if the response was not successful.
 
+Default Metrics
+---------------
+
+``Ping``
+~~~~~~~~
+
++--------------------+----------------------------------------------------------------+
+|  **measurement**:  |                           ``ping``                             |
++--------------------+----------------------------------------------------------------+
+|  **value type**:   |          ``int`` (reachable / loss), ``float`` (rtt)           |
++--------------------+----------------------------------------------------------------+
+|    **fields**:     | ``reachable``, ``loss``, ``rtt_min``, ``rtt_max``, ``rtt_avg`` |
++--------------------+----------------------------------------------------------------+
+| **configuration**: |                           ``ping``                             |
++--------------------+----------------------------------------------------------------+
+|    **charts**:     |             ``uptime``, ``packet_loss``, ``rtt``               |
++--------------------+----------------------------------------------------------------+
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/master/docs/uptime.png
+  :align: center
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/master/docs/downtime.png
+  :align: center
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/master/docs/rtt.png
+  :align: center
+
+``Traffic``
+~~~~~~~~~~~
+
++--------------------+-------------------------------------------------+
+|  **measurement**:  |             ``<interface_name>``                |
++--------------------+-------------------------------------------------+
+|  **value type**:   |                    ``int``                      |
++--------------------+-------------------------------------------------+
+|    **fields**:     |          ``<stat>`` (rx_bytes / tx_bytes)       |
```

Comment:
I think you can remove `<stat>`

Reply:
I have separated the two traffic metrics info (which is the case currently). We can change this once these two metrics get merged.